File: test_archives/cpp_tests/batch1/test_function6.cpp
#include "../catch.hpp"
#include "../functions/function6.cpp"

TEST_CASE("Test function6 with parameters 1, 'hello world', ['foo', 'bar']", "[function6]"){
  REQUIRE(function6(1, "hello world", std::vector<std::string>{"foo", "bar"}) == 6);
}

TEST_CASE("Test function6 with parameters 2, 'how are you', ['baz', 'qux']", "[function6]"){
  REQUIRE(function6(2, "how are you", std::vector<std::string>{"baz", "qux"}) == 8);
}

File: test_archives/py_tests/batch1/test_function6.py
from functions.function6 import function6
import pytest

def test_function6_1():  
  assert function6(1, 'hello world', ['foo', 'bar']) == 6

def test_function6_2():  
  assert function6(2, 'how are you', ['baz', 'qux']) == 8